### PR TITLE
Chore: Adjusts facets tokens

### PR DESCRIPTION
**Pull request by eduardoformiga**
## What's the purpose of this pull request?

Aims to adjust the tokens' names in `faces.module.scss` file.

## Checklist

<em>You may erase this after checking them all ;)</em>

**Changelog**
- [x] Added an entry in the `CHANGELOG.md` at the beginning of its due section. [The latest version should comes first](https://keepachangelog.com/en/1.0.0/#:~:text=The%20latest%20version%20comes%20first.).
- [x] Added the PR number with the PR link at the entry in the `CHANGELOG.md`. E.g., *New items in the `pull_request_template.md` ([#4](https://github.com/vtex-sites/nextjs.store/pull/4))* 

**PR Description**
- [x] Added a label according to the PR goal - `Breaking change`, `Enhancement`, `Bug` or `Chore`.
- [x] Added the component, hook, or pathname in-between backticks (``) *- If applicable*. E.g., *`ComponentName` component*.
- [x] Identified the function or parameter in the PR *- If applicable*. E.g., *`useWindowDimensions` hook*.

**Documentation**
- [x] PR description
- [x] Added to/Updated the Storybook - *if applicable*.
- [ ] For documentation changes, ping @ carolinamenezes, @ PedroAntunesCosta or @ Mariana-Caetano to review and update.

### CODE REVIEW

File: src/components/search/Filter/facets.module.scss [hunk 1]
```diff
@@ -10,17 +10,17 @@
   --fs-facets-title-margin-bottom                        : var(--fs-spacing-0);
 
   // Accordion
-  --fs-facets-accordion-notebook-border-width            : 1px;
-  --fs-facets-accordion-notebook-border-color            : var(--fs-border-color-light);
-  --fs-facets-accordion-notebook-border-radius           : var(--fs-border-radius);
+  --fs-facets-accordion-border-width-notebook            : 1px;
```
Comment (hellofanny) [inline src/components/search/Filter/facets.module.scss:13]:
```suggestion
  --fs-facets-accordion-border-width-notebook            : var(--fs-border-width);
```

😅 Can you update all the occurrences? thanks

File: CHANGELOG.md [hunk 2]
```diff
@@ -30,6 +30,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Updates `facets` tokens breakpoint name ([#150](https://github.com/vtex-sites/nextjs.store/pull/150))
```
Comment (hellofanny) [inline CHANGELOG.md:33]:
```suggestion
- Updates `facets` breakpoints tokens ([#150](https://github.com/vtex-sites/nextjs.store/pull/150))
```

**Review by hellofanny — APPROVED** [review]
Uhul! LGTM! 🚀

File: src/components/search/Filter/facets.module.scss (hunk 1 above)
Comment (eduardoformiga) [reply src/components/search/Filter/facets.module.scss]: done [here](https://github.com/vtex-sites/nextjs.store/pull/150/commits/7dece2f2f7edf0e096ec3edeb118eca4d626e00e).

**Pull request merged**